### DISCUSSION TITLE
Suggestion to rename project?

### DISCUSSION THREAD

**Discussion by rencire** [pluto-lang/pluto#238] [Ideas]
This project looks very cool. I was thinking maybe it might be better to rename this project to something other than pluto-lang? On search engines, when searching for "pluto lang", the top results always point to the Lua dialect (https://github.com/PlutoLang/Pluto)

**Comment by jianzs** [comment]
Thank you for your interest in the project! Currently, pluto-lang is still in its early stages. As it matures, I believe it will eventually achieve better SEO rankings, surpassing the other PlutoLang :) Of course, if you have any fantastic name suggestions, feel free to share them!
